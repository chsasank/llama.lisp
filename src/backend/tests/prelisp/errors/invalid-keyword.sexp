(define ((main int))
    ,(macro_func posarg1 posarg2 (#:kwarg1 val1) (#:kwarg2 val2.1 val2.2)))
