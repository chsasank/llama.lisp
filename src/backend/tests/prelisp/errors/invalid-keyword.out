AssertionError: Invalid keyword argument: ['#:kwarg2', 'val2.1', 'val2.2']
